setup.cfg

```diff
@@ -113,16 +113,21 @@ ignore = D202,D205,D302,D400,D105,D107,F401
 exclude = extern,sphinx,*parsetab.py,conftest.py,docs/conf.py,setup.py
 max-line-length = 99
 
-
-
 [coverage:run]
 omit =
     ci-helpers/*
     */tests/*
     plasmapy/setup_package.py
     plasmapy/version.py
+
 [coverage:report]
 exclude_lines =
     coverage: ignore
     ImportError
     ModuleNotFoundError
+
+[isort]
+# Set sorting of imports to be consistent with black formatting
+line_length=88
```

Comment:
We should probably make this consistent with the `max-line-length` for [`pycodestyle`](https://github.com/PlasmaPy/PlasmaPy/blob/10287a3aeaf82fa1b1df2036e0971f1acdbd8ee8/setup.cfg#L92) and [`flake8`](https://github.com/PlasmaPy/PlasmaPy/blob/10287a3aeaf82fa1b1df2036e0971f1acdbd8ee8/setup.cfg#L114), which are currently set to 99.

Reply:
```suggestiion
line_length=99 
```